roles/vald/templates/testnet/systemd.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description={{ service_identifier }}
+
+[Service]
+User={{ service_identifier }}
+Group={{ service_identifier }}
+WorkingDirectory={{ vald_home_dir }}
+EnvironmentFile={{ vald_environment_secrets_file_path }}
+ExecStart=/usr/bin/sh -c 'echo $KEYRING_PASSWORD | {{ vald_bin_dir }}/{{ service_identifier }} vald-start \
+    --home {{ vald_home_dir }} \
```

Comment:
this should be axelar home

Reply:
lets add another variable in the inventory file vald_axelar_home: /opt/axelar and we can use that here